tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -3227,6 +3231,46 @@ public function theRoomNeedToHaveAnAvatarWithStatusCode(string $identifier, int
 		$this->assertStatusCode($this->response, $statusCode);
 	}
 
+	/**
+	 * @When /^the room "([^"]*)" has an svg as avatar with (\d+)(?: \((v1)\))?$/
+	 */
+	public function theRoomNeedsToHaveAnSvgAvatarWithStatusCode(string $identifier, int $statusCode, string $apiVersion = 'v1'): void {
```

Comment:
Optional at this point:
```suggestion
	public function theRoomNeedsToHaveASvgAvatarWithStatusCode(string $identifier, int $statusCode, string $apiVersion = 'v1'): void {
```